cvat-sdk/gen/generate.sh

```diff
@@ -8,7 +8,7 @@ set -e
 
 GENERATOR_VERSION="v6.0.1"
 
-VERSION="2.4.0"
+VERSION="2.4.1"
```

Comment:
I recommend to call this version `2.4.0.post1` to avoid confusion with possible `2.4.1` in future.